src/components/ImportTokenButton.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react'
+import { useTranslation } from 'react-i18next'
+import { StyleProp, StyleSheet, TextStyle } from 'react-native'
+import { AnalyticsEventType } from 'src/analytics/Properties'
+import Times from 'src/icons/Times'
+import { navigateBack } from 'src/navigator/NavigationService'
+import { TopBarIconButton, TopBarTextButton } from 'src/navigator/TopBarButton'
+import { Colors } from 'src/styles/colors'
+import { typeScale } from 'src/styles/fonts'
+
+interface Props {
+  onCancel?: () => void
+  style?: StyleProp<TextStyle>
+  eventName?: AnalyticsEventType
+  buttonType?: 'text' | 'icon'
```

Comment:
these don't make sense to me.
1. what does "onCancel" even mean when you are importing a token? This button is only used in one place-- does it even need a configurable onPress prop?
2. Again, the button's only in one place in the designs-- why add a style prop?
3. For the same reason, why does it need a configurable analytics event name?
4. Same reason, why allow it to be text or icon if it's only going to be text?